cypress/e2e/awx/resources/workflowJobTemplateSurveys.cy.ts

```diff
@@ -12,7 +12,7 @@ describe('Workflow Job Templates Surveys', function () {
     cy.awxLogin();
   });
 
-  describe('WFJT Surveys: Create, Edit and Delete', function () {
```

Comment:
For the previous skipped ones we add a flag such ```// FLAKY_06_13_2024``` to allow to search, and identify when that flaky was introduced.

Reply:
Oh great, I'll add that, thank you